sdk/servicebus/Azure.Messaging.ServiceBus/src/Azure.Messaging.ServiceBus.csproj

```diff
@@ -18,6 +18,7 @@
     <PackageReference Include="Azure.Identity" />
     <PackageReference Include="Microsoft.Azure.Amqp" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" />
+    <PackageReference Include="System.Collections.Immutable" />
```

Comment:
I don't see anywhere this is actually used. Did you forget to commit ServiceBusReceivedMessage?

Reply:
As a friend of mine used to say - sucks to be an idiot. Sorry, updated.

Reply:
> As a friend of mine used to say - sucks to be an idiot. Sorry, updated.

Hehe, he doesn't say it anymore?

Reply:
Nope. He's no longer around to say that.

Reply:
Ah, sorry about that.

Reply:
All good. Just don't continue his tradition 🙂